client/src/components/ui/Container.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+    import { ContainerType } from '../types';
+
+    export let ty: ContainerType;
+</script>
+<section class={ty}>
+    <slot></slot>
+</section>
+
+<style>
+    section {
+        margin: var(--spacing-small);
+        padding: var(--spacing-large);
+        display: flex;
+        flex-direction: column;
+        gap: var(--spacing-small);
+        border-radius: var(--border-radius);
+    }
+
+    .enumeration {
+        background-color: var(--dashboard-sidedrawer);
+        box-shadow: var(--spacing-small) var(--spacing-small) var(--spacing-small) 0px var(--shadow-color);
+    }
+
+    .divider {
+        border: var(--spacing-tiny) solid var(--secondary-color);
+        box-shadow: var(--spacing-normal) var(--spacing-normal) var(--spacing-small) 0px var(--shadow-color);
+        background-color: white;
+    }
```

Comment:
Zero is actually unitless in CSS. 👌 
```suggestion
    .enumeration {
        background-color: var(--dashboard-sidedrawer);
        box-shadow: var(--spacing-small) var(--spacing-small) var(--spacing-small) 0 var(--shadow-color);
    }

    .divider {
        border: var(--spacing-tiny) solid var(--secondary-color);
        box-shadow: var(--spacing-normal) var(--spacing-normal) var(--spacing-small) 0 var(--shadow-color);
        background-color: white;
    }
```